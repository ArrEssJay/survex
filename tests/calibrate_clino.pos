( Easting, Northing, Altitude )
(    0.00,     0.00,     0.00 ) 1
(    0.00,     1.43,     0.00 ) 2
(    0.00,    10.00,     0.00 ) 2_
(    0.00,    10.00,     0.00 ) 2__
(    0.00,     1.43,   -10.00 ) d
(    0.00,     1.43,   -10.00 ) d_
(    0.00,     1.43,   -10.00 ) d__
(    0.00,     0.71,    10.00 ) u
(    0.00,     0.71,    10.00 ) u_
(    0.00,     0.71,    10.00 ) u__
(   -0.17,     1.43,    10.00 ) w
(   -0.17,     1.43,    10.00 ) w_
(   -0.17,     1.43,    10.00 ) w__
(   -0.17,     1.43,   -10.00 ) x
(   -0.17,     1.43,   -10.00 ) x_
(   -0.17,     1.43,   -10.00 ) x__
(  -10.00,     1.43,    -0.17 ) y
(    9.95,     1.43,     1.05 ) z
