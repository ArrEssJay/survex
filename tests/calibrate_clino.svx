; pos=yes warn=16
*begin f
*fix 1 0 0 0
1 2 10.00 000 00
1 u 10.00 090 +90
2 u 10.00 090 90
2 d 10.00 090 -90
; these should warn
2 w 10.00 090 91
2 x 10.00 090 -91
2 y 10.00 090 181
2 z 10.00 090 366
*calibrate clino 90
1 2_ 10.00 000 90
1 u_ 10.00 090 +180
2 u_ 10.00 090 180
2 d_ 10.00 090 0
; these should warn
2 w_ 10.00 090 181
2 x_ 10.00 090 -1
*calibrate clino 90 -1
1 2__ 10.00 000 90
1 u__ 10.00 090 0
2 u__ 10.00 090 0
2 d__ 10.00 090 180
; these should warn
2 w__ 10.00 090 -1
2 x__ 10.00 090 181
*end f
*begin b
*data normal from to tape compass backclino
*fix 1 0 0 0
1 2 10.00 000 00
1 u 10.00 090 -90
2 u 10.00 090 -90
2 d 10.00 090 +90
; these should warn
2 w 10.00 090 -91
2 x 10.00 090 91
2 y 10.00 090 -181
2 z 10.00 090 -366
*calibrate backclino 90
1 2_ 10.00 000 90
1 u_ 10.00 090 0
2 u_ 10.00 090 0
2 d_ 10.00 090 180
; these should warn
2 w_ 10.00 090 -1
2 x_ 10.00 090 181
*calibrate backclino 90 -1
1 2__ 10.00 000 90
1 u__ 10.00 090 180
2 u__ 10.00 090 180
2 d__ 10.00 090 0
; these should warn
2 w__ 10.00 090 181
2 x__ 10.00 090 -1
*end b
