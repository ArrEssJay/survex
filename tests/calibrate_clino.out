./calibrate_clino.svx:7: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:8: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:9: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:10: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:17: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:18: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:25: warning: Clino reading over 90 degrees (absolute value)
./calibrate_clino.svx:26: warning: Clino reading over 90 degrees (absolute value)

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Solving one equation...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 18 survey stations, joined by 20 legs.
There are 3 loops.
Total length of survey legs =  200.00m ( 191.58m adjusted)
Total plan length of survey legs =   50.99m
Total vertical length of survey legs =  151.21m
Vertical range = 20.00m (from u at 10.00m to d__ at -10.00m)
North-South range = 10.00m (from 2__ at 10.00m to 1 at 0.00m)
East-West range = 19.94m (from z at 9.95m to y at -10.00m)
  13 1-nodes.
   3 2-nodes.
   1 6-node.
   1 15-node.
Done.
There were 8 warning(s).
